File: Algos/bubble_sort.py
# iterates over given list as many times as it takes to sort from least to greatest value

x = [5,3,1,9,6,2,8,7,4]

def bubbleSort(input_list):
    for i in range(len(input_list) - 1):
        for j in range(len(input_list) - 1 - i):
            if input_list[j] > input_list[j + 1]:
                # print(i,j,input_list[j], input_list[j+1])
                input_list[j], input_list[j + 1] = input_list[j + 1], input_list[j]
    return input_list

print(bubbleSort(x))

File: Algos/bubble_sort.js
// iterates over given list as many times as it takes to sort from least to greatest value

let x = [5, 3, 1, 9, 6, 2, 8, 7, 4];

function bubbleSort(input_arr) {
    for (let i = 0; i < input_arr.length; i++) {
        for (let j = 0; j < input_arr.length - i; j++) {
            if (input_arr[j] > input_arr[j + 1]) {
                let swapTemp = input_arr[j];
                input_arr[j] = input_arr[j + 1];
                input_arr[j + 1] = swapTemp;
            }
        }
    }
    return input_arr
};

console.log(bubbleSort(x));